packages/lab/src/table/Table.tsx

```diff
@@ -0,0 +1,67 @@
+import { makePrefixer } from "@salt-ds/core";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+import { clsx } from "clsx";
+import { type ComponentPropsWithoutRef, forwardRef } from "react";
+
+import tableCss from "./Table.css";
+
+export const TableVariantValues = ["primary", "secondary", "tertiary"] as const;
+export type TableVariant = (typeof TableVariantValues)[number];
+
+export interface TableProps extends ComponentPropsWithoutRef<"table"> {
+  /**
+   * Styling variant. Defaults to "primary".
+   * @default primary
+   */
+  variant?: TableVariant;
+  /**
+   * Divider styling variant. Defaults to "tertiary";
+   * @default secondary
+   */
+  divider?: "primary" | "secondary" | "tertiary" | "none";
+  /**
+   * Zebra styling. Applies variant to every other row.
+   * @default undefined
+   */
+  zebra?: TableVariant;
+}
+
+export const withTableBaseName = makePrefixer("saltTable");
+
+export const Table = forwardRef<HTMLTableElement, TableProps>(function Table(
+  {
+    children,
+    className,
+    variant = "primary",
+    divider = "tertiary",
+    zebra = undefined,
```

Comment:
```suggestion
    zebra,
```

The default isn't really needed